toolchain/papiTest.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2018 IBM.
+# Author: Umesh S <umess1@linux.vnet.ibm.com>
+
+
+import os
+from avocado.utils import process
+from avocado import Test
+from avocado import main
+from avocado.utils import git, build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class papiTest(Test):
+
+    '''
+    This testcase make use of testsuite provided by the
+    source package, source files are downloaded and compiled
+    '''
+
+    def setUp(self):
+
+        sm = SoftwareManager()
+
+        for package in ['make', 'gcc']:
+            if not sm.check_installed(package) and not sm.install(package):
+                self.cancel("%s is needed for the test to be run" % package)
+
+        git.get_repo('https://github.com/arm-hpc/papi.git',
+                     destination_dir=self.srcdir)
+        self.path = os.path.join(self.srcdir + '/src')
+
+        os.chdir(self.path)
+        process.run('./configure', shell=True)
+        build.make(self.path)
+
+    def test(self):
+        '''
+        Runs the tests
+        '''
+
+        process.run('./run_tests.sh -a 2>&1 | tee %s/output.log' %
```

Comment:
Wouldn't `process.run()` return object contain the output, which can be used here ?